pkg/mount/switch_root_linux.go

```diff
@@ -2,13 +2,10 @@
 // Use of this source code is governed by a BSD-style
```

Comment:
We thought we needed this for nichrome, but never used it. Is there a case to be made for deleting it?

Reply:
No, FB uses switch_root.

Reply:
wow, that's good to hear.